crates/workspace/src/workspace.rs

```diff
@@ -4675,6 +4683,41 @@ impl Workspace {
     pub fn zoomed_item(&self) -> Option<&AnyWeakView> {
         self.zoomed.as_ref()
     }
+
+    pub fn activate_next_window(&mut self, cx: &mut ViewContext<Self>) {
+        let Some(current_window_id) = cx.active_window().map(|a| a.window_id()) else {
+            return;
+        };
+        let Some(next_window) = cx
+            .windows()
+            .iter()
+            .cycle()
+            .skip_while(|window| window.window_id() != current_window_id)
+            .nth(1)
+            .cloned()
```

Comment:
> `.cloned()` 

Do we need that here and below, given that all we do is `.update` and use its context?